template/docker/Dockerfile.j2

```diff
@@ -1,31 +1,90 @@
+# syntax=docker/dockerfile:1.10.0@sha256:865e5dd094beca432e8c0a1d5e1c465db5f998dca4e439981029b3b81fb39ed5
 # =============
 # This file is automatically generated from the templates in stackabletech/operator-templating
 # DON'T MANUALLY EDIT THIS FILE
 # =============
+# NOTE: The syntax directive needs to be the first line in a Dockerfile
+# https://docs.docker.com/build/checks/#fail-build-on-check-violations
+# check=error=true
 FROM oci.stackable.tech/sdp/ubi9-rust-builder AS builder
 
 FROM registry.access.redhat.com/ubi9/ubi-minimal AS operator
 
 ARG VERSION
 ARG RELEASE="1"
 
-LABEL name="Stackable Operator for {[ operator.pretty_string }]" \
-  maintainer="info@stackable.tech" \
-  vendor="Stackable GmbH" \
-  version="${VERSION}" \
-  release="${RELEASE}" \
-  summary="Deploy and manage {[ operator.pretty_string }] clusters." \
-  description="Deploy and manage {[ operator.pretty_string }] clusters."
+# These are chosen at random and are this high on purpose to have very little chance to clash with an existing user or group on the host system
+ARG STACKABLE_USER_GID="574654813"
+ARG STACKABLE_USER_UID="782252253"
```

Comment:
@Techassi can you please give some details on the high ID numbers on some Linux systems?

Reply:
I did some research on this and it seems as if "in the old days" only 16 bit UIDs were supported but that restriction seems to have been lifted a long time ago.

I'd say we're good with having this in.

https://unix.stackexchange.com/questions/191663/any-concerns-with-using-high-uid-numbers-3000-on-rhel5

Reply:
Works for me